### PR TITLE
Header

### PR DESCRIPTION
**Pull request by maxim-maidan**
Header added.

### CODE REVIEW

File: sources/scss/header.scss [hunk 1]
```diff
@@ -0,0 +1,259 @@
+header{
```
Comment (NazarMaidan) [inline sources/scss/header.scss]: use .class only

File: sources/scss/header.scss [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+
+	padding:40px 108px 40px 185px;
+
+	display:flex;
+	flex-direction:row;
+	justify-content:space-between;
+
+	border:1px solid black;
+
+	#logo{
```
Comment (NazarMaidan) [inline sources/scss/header.scss]: use .class only

File: sources/scss/header.scss [hunk 3]
```diff
@@ -0,0 +1,259 @@
+header{
+	width:100;
```
Comment (NazarMaidan) [inline sources/scss/header.scss]: width:100 (what?)   px/vw/%/em/rem

File: sources/scss/header.scss [hunk 4]
```diff
@@ -0,0 +1,259 @@
...
+	height:auto;
+
+	padding:40px 108px 40px 185px;
+
+	display:flex;
+	flex-direction:row;
+	justify-content:space-between;
+
+	border:1px solid black;
```
Comment (NazarMaidan) [inline sources/scss/header.scss]: black is a color - use it as const in color.scss file
use #hex for colors instead of names 
$black: #000

File: sources/scss/header.scss [hunk 5]
```diff
@@ -0,0 +1,259 @@
...
+		flex-direction:row;
+		justifu-content:space-between;
+
+		&__element{
+			height:44px;
+			width:auto;
+
+			padding:14px 18px;
+
+			font-size:16px;
```
Comment (NazarMaidan) [inline sources/scss/header.scss]: font-size should be in "em" instead of "px"

and use const for fz value

File: sources/scss/header.scss [hunk 6]
```diff
@@ -0,0 +1,259 @@
...
+	#logo{
+		width:50px;
+		height:50px;
+		&:hover{
+			cursor: pointer;
+		}
+	}
+	.nav{
+		width:auto;
+		height:44px;
```
Comment (NazarMaidan) [inline sources/scss/header.scss]: the .nav class shouldn't have height. The children give an height to it

File: sources/scss/header.scss [hunk 7]
```diff
@@ -0,0 +1,259 @@
...
+				}
+			}
+			.menu__btn{
+				display: none;
+			    align-items: center;
+			    position: absolute;
+			    top: 40px;
+			    right: 7.5%;
+
+			    width: 50px;
```
Comment (NazarMaidan) [inline sources/scss/header.scss]: 50px => use as const

File: sources/scss/header.scss [hunk 8]
```diff
@@ -0,0 +1,259 @@
...
+			    height: 50px;
+
+			    padding: 0;
+
+			    cursor: pointer;
+			    z-index: 1000;
```
Comment (NazarMaidan) [inline sources/scss/header.scss]: no need give such huge z-index

File: sources/scss/header.scss [hunk 9]
```diff
@@ -0,0 +1,259 @@
...
+
+				width: 200px;
+				height: 100%;
+
+				margin: 0;
+				padding: 100px 0 50px 0;
+
+				list-style: none;
+
+				background-color: #ECEFF1;
```
Comment (NazarMaidan) [inline sources/scss/header.scss]: const

File: sources/scss/header.scss [hunk 10]
```diff
@@ -0,0 +1,259 @@
...
+				box-shadow: 1px 0px 6px rgba(0, 0, 0, .2);
```
Comment (NazarMaidan) [inline sources/scss/header.scss]: const

File: sources/scss/header.scss [hunk 11]
```diff
@@ -0,0 +1,259 @@
...
+
+				z-index:900;
+				
+				&__wrapper{
+					.menu__item{
+						display: block;
+						padding: 12px 24px;
+						margin: 0 auto;
+						text-align: left;
+						color: #333;
```
Comment (NazarMaidan) [inline sources/scss/header.scss]: const

File: sources/scss/header.scss [hunk 12]
```diff
@@ -0,0 +1,259 @@
...
+
+						font-family: 'Roboto', sans-serif;
+						font-size: 20px;
```
Comment (NazarMaidan) [inline sources/scss/header.scss]: em and const

File: sources/scss/header.scss [hunk 13]
```diff
@@ -0,0 +1,259 @@
...
+				display: block;
+				position: fixed;
+				visibility: hidden;
+				left: 0;
+				top: 0;
+
+				width: 100%;
+				height: 100%;
+
+				background-color: rgba(0, 0, 0, 0.54);
```
Comment (NazarMaidan) [inline sources/scss/header.scss]: const

File: sources/scss/header.scss [hunk 14]
```diff
@@ -0,0 +1,259 @@
...
+			}
+		}
+
+		@media screen and (min-width:840px){
+			&{
+				display:none;
+			}
+		} 
+	}
+	@media screen and (max-width: 1030px){
```
Comment (NazarMaidan) [inline sources/scss/header.scss]: you are using wrong breakpoints 
image

File: sources/scss/colors.scss [hunk 15]
```diff
@@ -0,0 +1,4 @@
+$green:#1abe84;
+$orange: #fb8f1d;
+$dark-blue:#202336;
+$gray:#c4c9d5;
```
Comment (NazarMaidan) [inline sources/scss/colors.scss]: add all colors that you can find in Figma

add black, white

File: sources/scss/body.scss [hunk 16]
```diff
@@ -0,0 +1,129 @@
...
+
+
+input,
+textarea {
+  font-family: inherit;
+}
+
+input::-ms-clear {
+  display: none;
+}
```
Comment (NazarMaidan) [inline sources/scss/body.scss]: it shouldn't be here. all those styles look like normalize.css

File: sources/scss/body.scss [hunk 17]
```diff
@@ -0,0 +1,129 @@
...
+
+  background-color:white;
+
+}
+button:click{
+  
+}
+button:active{
+  transform:translateY(1px);
+}
```
Comment (NazarMaidan) [inline sources/scss/body.scss]: you have to add a separate file for buttons

File: sources/scss/body.scss [hunk 18]
```diff
@@ -0,0 +1,129 @@
...
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  font-size: inherit;
+  font-weight: 400;
+}
```
Comment (NazarMaidan) [inline sources/scss/body.scss]: either normalize.css

File: index.html [hunk 19]
```diff
@@ -0,0 +1,51 @@
...
+
+    <link rel="stylesheet" href="css/index.css" type="text/css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/d6e76c230a.css">
+
+</head>
+<body>
+	<header>
+		<img id="logo" src="img/logo.png">
+		<div class="nav">
+			<div class="nav__element nav__element-active">Home</div>
```
Comment (NazarMaidan) [inline index.html]: `nav__element-active` - modifficator 
should be `nav__element--active`

File: index.html [hunk 20]
```diff
@@ -0,0 +1,51 @@
...
+			<div class="nav__element">Destinations</div>
+			<div class="nav__element">About</div>
+			<div class="nav__element">Partner</div>
```
Comment (NazarMaidan) [inline index.html]: use `item` instead of `element`

File: index.html [hunk 21]
```diff
@@ -0,0 +1,51 @@
...
+		    			<li><a class="menu__item" href="#">Destinations</a></li>
+		            	<li><a class="menu__item" href="#">About</a></li>
+		            	<li><a class="menu__item" href="#">Partner</a></li>
+		        	</div>
+		        	<div class="menu__buttons">
+		        		<button class="login">Login</button>
+						<button class="register">Register</button>	
+		        	</div>
+		        </ul>
+		    </div>
```
Comment (NazarMaidan) [inline index.html]: no need to have 2 separate menus
use the same? but for different screen-width use different styles

File: index.html [hunk 22]
```diff
@@ -0,0 +1,51 @@
...
+		</div>
+	</header>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
```
Comment (NazarMaidan) [inline index.html]: do not use jquery - it's already dead lib
use JS

File: index.html [hunk 23]
```diff
@@ -0,0 +1,51 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1"/>
+
+    <link rel="stylesheet" href="css/index.css" type="text/css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/d6e76c230a.css">
+
+</head>
+<body>
+	<header>
+		<img id="logo" src="img/logo.png">
+		<div class="nav">
```
Comment (NazarMaidan) [inline index.html]: `<div class="nav"> ... </div>` => `<nav class="nav"> ... </nav>`

File: .gitignore [hunk 24]
```diff
@@ -0,0 +1,3 @@
+
+css/header.css
+css/header.css
```
Comment (NazarMaidan) [inline .gitignore]: why twice?

File: index.html [hunk 25]
```diff
@@ -33,19 +33,17 @@
...
 		    			<li><a class="menu__item" href="#">Destinations</a></li>
 		            	<li><a class="menu__item" href="#">About</a></li>
 		            	<li><a class="menu__item" href="#">Partner</a></li>
-		        	</div>
-		        	<div class="menu__buttons">
+
 		        		<button class="login">Login</button>
 						<button class="register">Register</button>	
-		        	</div>
 		        </ul>
```
Comment (NazarMaidan) [inline index.html]: you need to delete it at all

you have your `<nav class="nav"> ... </nav> `  and if screen-width less then 768px you need to change styles of `.nav`

File: sources/scss/button.scss [hunk 26]
```diff
@@ -0,0 +1,51 @@
+@import"constants";
+button{
+  height:$item-height;
+  width:130px;
```
Comment (NazarMaidan) [inline sources/scss/button.scss]: set width/height => use padding for that

File: sources/scss/button.scss [hunk 27]
```diff
@@ -0,0 +1,51 @@
...
+  border: 1px solid $orange;
+
+  box-shadow: 0 1px $orange;
+
+  background-color:$white;
+
+}
+button:click{
+  
+}
```
Comment (NazarMaidan) [inline sources/scss/button.scss]: remove if you don't use it

File: sources/scss/header.scss [hunk 28]
```diff
@@ -60,18 +60,18 @@ header{
...
 			padding:0 0 0 12px;
 			
 			button{
 				margin:0;
 
-				font-size:16px;
+				font-size:$header-fs;
 				font-family: 'Roboto', sans-serif;				
 				font-weight:600;
 			}
```
Comment (NazarMaidan) [inline sources/scss/header.scss]: this style should be in button.scss file

File: sources/scss/normalize.scss [hunk 29]
```diff
@@ -1,20 +1,16 @@
-@import"colors";
+@import"constants";
```
Comment (NazarMaidan) [inline sources/scss/normalize.scss]: remove it. because you don't use any const here
